eip/EIPS/eip-4337.md

```diff
@@ -262,12 +262,13 @@ calling:
 3. if using an aggregator: `aggregator.validateUserOpSignature`.
 
 The operations differ in their opcode banning policy.
-In order to distinguish between them, there is a single call to the NUMBER opcode (`block.number`), used as a delimiter between the validation functions.
+In order to distinguish between them, there is a call to the NUMBER opcode (`block.number`), used as a delimiter between the validation functions.
 While simulating `userOp` validation, the client should make sure that:
 
 1. Neither call's execution trace invokes any **forbidden opcodes**
 2. The first (validateUserOp) call does not access _mutable state_ of any contract except the wallet itself and its deposit in the entry point contract. _Mutable state_ definition includes both storage and balance.
-3. The second (validatePaymasterUserOp) call does not access _mutable state_ of any contract except the paymaster itself.
+3. The second (validatePaymasterUserOp) call does not access _mutable state_ of any contract except the paymaster itself. The paymaster is also not allowed to make any state-change calls (SSTORE)
+   A bundler MAY whitelist specific paymaster addresses, and not perform the above storage limitations
```

Comment:
I'd say "not enforce" rather than "not perform"

Reply:
ok